Sources/ReleaseServer/ReleaseServer.WebApi/Repositories/ReleaseArtifactRepository.cs

```diff
@@ -200,6 +199,53 @@ public List<string> GetPlatforms(string productName, string version)
 
             return platforms.OrderBy(p => p).ToList();
         }
+        
+        public BackupInformationModel RunBackup()
+        {
+            var timeStamp = DateTime.UtcNow.ToString("yyyy-MM-dd-HH-mm-ss");
+            
+            string backupFileName = "backup_" + timeStamp + ".zip";
+            string backupArchiveFileName = Path.Combine(BackupRoot, backupFileName); 
+            
+            //Clear the backup folder first
+            DeleteFilesInPath(BackupRoot);
+            
+            //Create the backup -> zip the whole ArtifactRoot folder
+            ZipFile.CreateFromDirectory(ArtifactRoot, backupArchiveFileName);
```

Comment:
This method also includes the `/temp` directory inside the `ArtifactRoot` directory.
For the sake of correctness this should not be delivered with the backup.

Reply:
I fixed this issue with deleting the temp directory after storing the artifact (the temp directory is empty then).

See: 
https://github.com/Traeger-GmbH/release-server/blob/feature/implement-backup-mechanism/Sources/ReleaseServer/ReleaseServer.WebApi/Repositories/ReleaseArtifactRepository.cs#L74-77